formidable/forms/field_builder.py

```diff
@@ -253,8 +266,28 @@ class FormFieldFactory(object):
 
     def __init__(self, field_map=None):
         self.map = self.field_map.copy()
+        self.add_extra_fields()
         self.map.update(field_map or {})
 
+    def add_extra_fields(self):
+        # If the settings is not set, skip this step.
+        if not hasattr(settings, 'FORMIDABLE_EXTERNAL_FIELD_BUILDERS'):
+            return
+        extra = getattr(settings, 'FORMIDABLE_EXTERNAL_FIELD_BUILDERS', {})
+        # If empty or None
+        if not extra:
+            return
+
+        for key, classpath in extra.items():
+            module_name, class_name = classpath.rsplit('.', 1)
+            module = importlib.import_module(module_name)
+            field_builder_class = getattr(module, class_name)
```

Comment:
I'd like to move import part to the function in the utils (we will use it to get token in the future)